### PR TITLE
Cleanup of temp test files

### PR DESCRIPTION
**Pull request by goelakash**

### CODE REVIEW

File: test/test_regression_cli.py [hunk 1]
```diff
@@ -59,9 +59,12 @@ def setup_module():
 
 def teardown_module():
     """Cleanup temporary output files after testing and return to root directory"""
+    os.system("rm -r output*")
     os.system("rm output*")
```
Comment (henrykironde) [inline test/test_regression_cli.py]: remove the old ones

**Comment by henrykironde** [comment]
:+1:

File: test/test_regression_cli.py (hunk 1 above)
Comment (goelakash) [inline test/test_regression_cli.py]: I couldn't remove `output_json.csv` and `output_xml.csv` from these commands. Any idea how to remove those as well?

File: test/test_retriever.py [hunk 2]
```diff
@@ -36,6 +36,7 @@ def setup_module():
 def teardown_method():
     """Cleanup temporary output files after testing"""
     os.system("rm -r output*")
+    os.system("rm output*")
```
Comment (henrykironde) [inline test/test_retriever.py]: remove this too

**Comment by henrykironde** [comment]
try using os.remove()

**Comment by goelakash** [comment]
Didn't work.

**Comment by henrykironde** [comment]
okey there may be an error in the code in test/test_retriever.py   `def test_xml2csv()` i wiill check it out and clarify

**Comment by henrykironde** [comment]
is it currently working ?
if not : 

In test/test_retriever.py we trying to set up the testing by os/chir(dir_path)
but it looks like it does not change to that directory 

so what I think is `def test_xml2csv()` and `test_json2csv()` are run from the main retriever directory a go head and  try to use os.system("rm -r test/output*") in the Tear down 

to test the directory you can use this , change assert to` assert os.getcwd() ==""`

**Comment by henrykironde** [comment]
on more solution is to try deleting those files right before the assert statements.

**Comment by henrykironde** [comment]
@goelakash is this working as required?

**Comment by goelakash** [comment]
@henrykironde Let me get back to you in a few hours.

**Comment by goelakash** [comment]
@henrykironde Yes, it works now.

**Pull request merged**